### ISSUE TITLE
Modules Folder Clean-Up

### ISSUE THREAD

**Issue by willykaram** [NYC-Camp/website#198]
I just notices that our sites/all/modules folder is a bit jumbled. We don't have a separate folder for /contrib vs/custom modules. I would like to fix that quickly in the course of some other work. Does anyone object?

**Comment by ForestMars** [comment]
I'm pretty sure you'l need to rebuild the registry after you do that.

**Comment by rabellamy** [comment]
@willykaram do it!!!

**Comment by likewhoa** [comment]
remember this will require the rr drush filter enabled on @live so that the registry is rebuild properly. This should be tested in @test and QA fully by that make sure that all functionality is working. This issue also relates to my #16 issue which we should do first.

**Comment by willykaram** [comment]
If people still think we should do this, then I can do so late tonight. My concern is that it's going to make quite a few updates to everyone's branches, and we should ensure that everyone is on board with this.

I will implement this on a separate branch/multidev and test thoroughly before implementing on live. I think that this is pretty much the last day when we can do this. After today, I'd rather not undertake this change as it's too close to the camp.

FWIW, at this point, I think we could just leave it as is until after the camp. As this is really a behind the scenes change that we're making for best practices purposes, but that doesn't have any UX or other benefit for camp attendees.

Perhaps @esod and @mirie and others can also weigh in

**Comment by esod** [comment]
Don't do this. It's really not necessary. Leave the modules folder as it is. We can get to this after the camp.

**Comment by willykaram** [comment]
I agree with Eric's comment right now. Unless anyone strongly objects, let's defer on this until after the camp.
